src/objectivec/oga_internal.h

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Keep things we don't want to expose internal, mostly C++ things.

Reply:
I'm not an objective-c expert. can you explain how this keeps things internal? maybe we could add a comment in the file if that would help the next developer understand how it works as we rarely work on objective-c.

Reply:
Oh I didn't include the podspec here because I think it's not ready. The way to keep it internal is by tweaking the podspec.